app/Services/MapboxGeocodingService.php

```diff
@@ -26,4 +29,35 @@ public function getCoordinatesFromApi(string $cityName, string $countryName): ar
             return [];
         }
     }
+
+    public function getPlaceFromApi(string $lat, string $long): array
+    {
+        $token = env("MAPBOX_TOKEN");
+        $client = new Client();
+
+        try {
+            $response = $client->get(
+                "https://api.mapbox.com/geocoding/v5/mapbox.places/$long,$lat.json?access_token=$token",
+            );
+            $features = json_decode($response->getBody()->getContents(), true)["features"];
```

Comment:
Please use named arguments for values which don't describe themselves. (raw numbers, booleans, etc.).
`associative: true`

https://stitcher.io/blog/php-8-named-arguments

Reply:
I'm not sure how to use named arguments here. 
$features depends on coordinates' actual features, number of those is not consistent, as well as its' id schema
for example, Perth, Australia (115.861309,-31.9523123) look like that:
```
$features= {
[0] => { 'id': 'address', [...]},
[1] => { 'id': 'postcode, [...]},
[2] => { 'id': 'locality', [...]},
[3] => { 'id': 'region', [...]},
[4] => {'id': 'country', [...]}
}
```

when Abu Dhabi, UAE (54.3773438, 24.453884):
```
$features= {
[0] => { 'id': 'poi', [...]},
[1] => { 'id': 'postcode, [...]},
[2] => { 'id': 'place', [...]},
[3] => { 'id': 'district', [...]},
[4] => {'id': 'region', [...]},
[5] => {'id': 'country', [...]}
}
```

Reply:
I thought about this boolean (true) value in `json_decode` method:

before:
```php
$features = json_decode($response->getBody()->getContents(), true)["features"];
```
after:
```php
$features = json_decode($response->getBody()->getContents(), associative: true)["features"];
```